pkg/apis/flowcontrol/helpers.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flowcontrol
+
+import "sort"
+
+var _ sort.Interface = FlowSchemaSequence{}
+
+// FlowSchemaSequence holds sorted set of pointers to FlowSchema objects.
+// FlowSchemaSequence implements `sort.Interface`
+type FlowSchemaSequence []*FlowSchema
```

Comment:
we don't place helpers in the API packages.

Reply:
moved it to `pkg/apis/flowcontrol/util/helpers.go `